examples/check-downloaded-file-name-and-content/README.MD

```diff
@@ -0,0 +1,21 @@
+# Check the Name and the Contents of a Downloaded File
+
+**Test Code**: [index.js](index.js)
+
+This example shows how to check the name and the content of a file downloaded during a test.
+
+During the test, a link click starts a file download. To verify download success and obtain the name and the content of the file, a [RequestLogger](https://devexpress.github.io/testcafe/documentation/reference/test-api/requestlogger/) is attached to the test with the [t.requestHooks](https://devexpress.github.io/testcafe/documentation/reference/test-api/test/requesthooks.html) method.
```

Comment:
> a link click starts a file download

`a link click starts a file download` sounds like "a cannon shot starts the festival". Maybe it's just `During the test, TestCafe clicks the link that triggers file download.`

> obtain the name and the content of the file

`obtain the file name and content` - much shorter

Reply:
edited